Gemfile

```diff
@@ -16,14 +16,11 @@ gem 'activerecord-session_store'
 # Allowing for URI templates, for HTTP clients
 gem 'addressable', '~> 2.6'
 
-# Pagination
-gem "api-pagination"
-
 # Encrypt DB data at rest
 gem "attr_encrypted", "~> 3.1.0"
 
 # Integration with Matomo Piwik
-gem 'autometal-piwik', :require => 'piwik', git: "https://github.com/matomo-org/piwik-ruby-api.git", branch: "master"
+gem 'autometal-piwik', :require => 'piwik', git: "https://github.com/yachtcaptain23/piwik-ruby-api.git", branch: "feature/rails_6_support"
```

Comment:
any reason not to publish this under brave on github?

Reply:
That I can do. Will drive this.